src/steamship/agents/context.py

```diff
@@ -0,0 +1,39 @@
+from typing import Dict, Any, List, Optional
+
+from pydantic import BaseModel
+
+from steamship import Steamship
+from steamship.experimental.transports.chat import ChatMessage
+
+
+class ChatHistory(BaseModel):
+    chat_id: str = ""  # chat history is scoped by a chat_id. chat_id is unique across user_id's
+    messages: List[str] = []  # Represents a database that can select & sort messages
```

Comment:
I think it's worth it to make `Block` the one true message type. There's so much that it instantly resolves:

- How we store it
- How we support multiple mime types
- How we annotate things atop it
- Etc etc

For the rest of the world that doesn't speak in blocks (e.g. steamship-langchain) we just do whatever conversion is necessary in the adapter library.

Reply:
+1. str was just a shortcut for me to get it working. Going to copy code from Doug now to get multi-modal working in this PR.